src/core.jl

```diff
@@ -104,4 +104,27 @@ function match_keypoints(keypoints_1::Keypoints, keypoints_2::Keypoints, desc_1,
         end
     end
     matches
+end
+
+"""
+```
+grade = grade_matches(keypoints_1, keypoints_2, difference_method)
+```
+
+Returns a measure of similarity between keypoints in `keypoints_1` and `keypoints_2` where `difference_method`
+is the method used for computing difference between individual pair of keypoints.
+"""
+
+function grade_matches(K1::Keypoints, K2::Keypoints, diff::Function = (i,j)->(abs(i[1] - j[1]) + (abs(i[2] - j[2]))))
```

Comment:
```
@assert length(k1)==length(k2) "Keypoint lists are of different lengths."
@assert length(k1)==0 "Keypoint list is of size zero."
```
@timholy will `@assert` slow this down too much? Assuming this function might be used with a continuous video stream for tracking keypoints, we need it to be fast.